tools/botskills/src/utils/sanitizationUtils.ts

```diff
@@ -24,3 +24,25 @@ export function sanitizePath(path: string): string {
 export function wrapPathWithQuotes(path: string): string {
     return `"${path}"`;
 }
+
+/**
+ * @param endpoint URL of the remove manifest endpoint.
+ * @param inlineUtterances Value of the --inlineUtterances parameter.
+ * @returns Returns an endpoint based on the --inlineUtterances parameter.
+ */
+export function validateRemoteEndpoint(endpoint: string, inlineUtterances: boolean): string {
+    const paramName: string = 'inlineUtterancesSources';
```

Comment:
If the `paramName` is the one of the query, it should be `inlineTriggerUtterances`, check this [manifest](https://bfcalendarskill.azurewebsites.net/api/skill/manifest?inlineTriggerUtterances=false)
```suggestion
    const paramName: string = 'inlineTriggerUtterances';
```